tests/issueFields.json

```diff
@@ -0,0 +1,21 @@
+{
+    "fields": {
+        "project": {
+            "id": "11100",
+            "key": "MON"
+        },
+        "summary": "Brand new Test execution",
```

Comment:
IMHO: It seems odd to always have the same ticket name 🤔

Reply:
should i remove it ?